src/mixins/wallet.js

```diff
@@ -93,5 +106,48 @@ export default {
         'menubar=no,location=no,width=576,height=768'
       );
     },
+    async fetchFollowers() {
+      try {
+        await this.walletFetchFollowers();
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error(error);
+      }
```

Comment:
`fetchFollowers` is not a good name for a function that purpose is wrapping `walletFetchFollowers` with try catch
`fetchFollowersWithTryCatch` maybe, idk @nwingt any comment?

Reply:
I think we should called walletFetchFollowers directly if we there is no other action
Also, I don't think we should put `walletFetchFollowers` in wallet mixin, as this action is not commonly used